### PR TITLE
Added card switch to reduce cards height.

### PR DESCRIPTION
**Pull request by jc-hiroto**

### CODE REVIEW

**Review by swh00tw — COMMENTED** [review]
牛B

**Comment by jc-hiroto** [comment]
image
Will modify this button later.

**Review by swh00tw — APPROVED** [review]

**Comment by swh00tw** [comment]
> image Will modify this button later.

Done! I'm so 🐂 🍺

File: src/containers/HomeViewContainer.js [hunk 1]
```diff
@@ -194,28 +219,40 @@ function HomeViewContainer(props) {
...
+                      newsCard.map((item, index) => { // eslint-disable-line
+                        if (index === displayingCard){
+                          return (
+                            <motion.div key={index} initial={{ opacity: 0, y: -20 }} animate={{ opacity: 1, y: 0 }} exit={{ opacity: 0, y: 20 }} transition={{duration: 0.4}}>
+                              {item}
+                            </motion.div>
+                          )
+                        }
+                      })
+                    }
```
Comment (jc-hiroto) [inline src/containers/HomeViewContainer.js:245]: Maybe `const item = newsCard[index]` can also achieve the same result...?
or do you have additional considerations? (like framework limitations...etc?)

**Review by jc-hiroto — COMMENTED** [review]
@swh00tw Nice job w/ framer-motion! 💯

File: src/containers/HomeViewContainer.js (hunk 1 above)
Comment (swh00tw) [reply src/containers/HomeViewContainer.js]: My bad. Resolved!

**Pull request merged**